pages/Programmable-Android.md

```diff
@@ -0,0 +1,112 @@
+Programmable Android category contains tools for the user for automating their task to make their life easier.
+
+## Android Interfaces
+
+### Android Interfaces (Cargo)
+
+Recipe:
+
+This interface is used to store items from an Android when the script tells then to do so.
+This interface is also can be connected through a cargo network or with a hopper.
+
+### Android Interfaces (Fuel)
+
+Recipe:
+
+This interface is used to give Android the fuel from it when the script tells then to do so.
+This interface is also can be connected through a cargo network or with a hopper.
```

Comment:
I would argue that the Interfaces deserve their own page

Reply:
> Please look into branches:
> https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/about-branches#working-with-branches
> 
> This way, you can make multiple pull requests and keep the changes seperate from each other.
> It is best practice to keep a PR as small as possible, as it makes the reviewing a whole lot easier and changes can be applied to limited scopes.

ah fuck i forgot to make a branch. I'll revert it to placeholder only for this one. (the android page is still on concept, also will change it to their own individual type page)

Reply:
:+1: